lib/sensu/extensions/occurrences.rb

```diff
@@ -21,11 +38,11 @@ def description
       # @return [Array] containing filter output and status.
       def event_filtered?(event)
         check = event[:check]
-        occurrences = check[:occurrences] || 1
-        refresh = check[:refresh] || 1800
+        occurrences = str2int(check[:occurrences]) || 1
+        refresh = str2int(check[:refresh]) || 1800
```

Comment:
Yep. The str2int only handles strings or integers. It will fail on any other value (including empty). On the other hand, if you use `check[:occurrences].to_i`, it converts missing variables to 0 instead of falling back to the default (eg: ` || 1`).